File: AOJ/0257/railway_ticket.java
import java.util.Scanner;

class Main {
  void solve() {
    Scanner sc = new Scanner(System.in);
    int b1 = sc.nextInt();
    int b2 = sc.nextInt();
    int b3 = sc.nextInt();
    if ( b1 == 1 && b2 == 1 || b3 == 1 )
      System.out.println("Open");
    else
      System.out.println("Close");
  }
    
  public static void main(String[] args) {
    new Main().solve();
  }
}

File: AOJ/0257/railway_ticket.c
#include<stdio.h>

int main() {
  int b1, b2, b3;
  scanf("%d %d %d", &b1, &b2, &b3);
  if ( b1 && b2 || b3 )
    printf("Open\n");
  else
    printf("Close\n");
  return 0;
}
